client/py/README.md

```diff
@@ -0,0 +1,15 @@
+# Python client for All of Us Workbench Jupyter Notebooks
+
+Usage:
+
+  virtualenv venv
```

Comment:
Is this for local development? Or something you'd do in a notebook hosted in Firecloud?

Reply:
The `pip install` would happen in a notebook server Dockerfile. I just included the `virtualenv` part for anyone who wants to try it out now -- this README's contents will probably get replaced.